brevproduksjon/src/main/java/no/nav/foreldrepenger/fpformidling/brevproduksjon/mapper/avslagfp/AvslåttPeriodeMapper.java

```diff
@@ -95,14 +97,22 @@
     }
 
     private static AvslåttPeriode årsaktypeFra(TilkjentYtelsePeriode tilkjentYtelsePeriode,
+                                               List<TilkjentYtelsePeriode> tilkjentPeriodeListe,
```

Comment:
Alternativt til å ta inn liste med perioder her kunne vært å knekke tilkjent ytelse periodene tidligere i ForeldrepengerAvslagDokumentdataMapper. Uten å ha sett for mye på det så tror jeg det vil være mer oversiktlig?
Mulig man da får to helt like avslagsperioder i brevet?

Vi må tenke at det er mulig at de to overlappende uttaksperiodene kan ha forskjellig avslagsårsak, det vil vel fortsatt være feil med denne løsningen?

Reply:
Knekke uttaksperioder tidligere***

Reply:
Litt usikker på hva du mener her...? Kanskje vi kunne tatt en prat? Uttaksperioden vil bare ha en avslagsårsak, det er tilkjentperiodene som i dette tilfellet har knekk